packages/playwright-test/src/util.ts

```diff
@@ -307,16 +307,8 @@ function folderIsModule(folder: string): boolean {
   return require(packageJsonPath).type === 'module';
 }
 
-export function experimentalLoaderOption() {
-  return ` --no-warnings --experimental-loader=${url.pathToFileURL(require.resolve('@playwright/test/lib/transform/esmLoader')).toString()}`;
-}
-
-export function envWithoutExperimentalLoaderOptions(): NodeJS.ProcessEnv {
-  const substring = experimentalLoaderOption();
-  const result = { ...process.env };
-  if (result.NODE_OPTIONS)
-    result.NODE_OPTIONS = result.NODE_OPTIONS.replace(substring, '').trim() || undefined;
-  return result;
+export function experimentalLoaderOptions() {
```

Comment:
Shouldn't we also use this in `processHost` when spawning workers? How do our tests pass?

Reply:
main process restarts with it, so all it's child processes inherit the same NODE_OPTIONS.